akd/src/lib.rs

```diff
@@ -18,8 +18,12 @@
 //! Keys can also be updated to be associated with different values. Each batch of updates to these key-value
 //! mappings are associated with an epoch along with a commitment to the database of entries at that point in time.
 //! The server that controls the database can use this library to generate proofs of inclusion to clients that wish
-//! to query entries in the database. These proofs can be _verified_ by a client against the corresponding
+//! to query entries in the database. Note that we are not just querying entries; we are querying mappings from labels
+//! to values in the database. These proofs can be _verified_ by a client against the corresponding
 //! commitment to the database. We can think of this data structure intuitively as a _verifiable dictionary_.
+//! Also, this isn’t enough for a full key transparency solution. We still need a way for clients to know
```

Comment:
First, let's leave a newline between the previous text and this text, so as to represent a logical separator of content.

Then, let's change the text to say:

"This library can be used as part of a key transparency system to generate commitments and serve proofs for a database of public keys. However, note that a full key transparency solution still needs to provide a way for clients to ensure that they are receiving the same commitment for each database epoch. This is outside of the scope of this library and must be handled separately."